internal/controllers/clusterextension_controller.go

```diff
@@ -206,6 +207,11 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1.Cl
 	installedBundle, err := r.InstalledBundleGetter.GetInstalledBundle(ctx, ext)
 	if err != nil {
 		setInstallStatus(ext, nil)
+		var saerr *tokengetter.ServiceAccountNotFoundError
+		if errors.As(err, &saerr) {
+			l.Info("is a SAError")
```

Comment:
Could we improve the wording and explanation here? What exactly is the error, and under what circumstances does it occur?

As a cluster admin or extension admin, I want to clearly understand why and when an expected error happens so that I can handle the scenario more effectively.